app/assets/javascripts/modules/download_panel.js

```diff
@@ -22,14 +22,14 @@
         .append('<a class="download-link" href="' + imageData['@id'] +
         '/full/!' + disableDownloadWidthCutoff + ',' +
         disableDownloadWidthCutoff +
-        '/0/default.jpg" download>Download Thumbnail</a>'));
+        '/0/default.jpg?download=true" download>Download Thumbnail</a>'));
```

Comment:
I noticed that we missed this one.  I assume we want the `download=true` for thumbnails as well as the other downloadable derivatives?